docs/preview/03-Features/secret-store/create-new-secret-provider.md

```diff
@@ -181,8 +165,48 @@ namespace Application.Controllers
 }
 ```
 
-### Adding secret name mutation before looking up secret
+### Adding secret versions to your secret provider
+When your secret storage location supports secret versions, you could consider adapting your secret provider to support these secret versions.
```

Comment:
```suggestion
When your secret storage location supports versioned secrets, you could consider adapting your secret provider to support retrieving secrets by version.
```
Maybe this makes it a bit more clear ?

Reply:
It's not 'by version' that they are requested, but by amount.
`When your secret storage location supports versioned secrets, you could consider adapting your secret provider to support retrieving multiple versions of a secret.`

Reply:
Changed in https://github.com/arcus-azure/arcus.security/pull/358/commits/b9fbf2f85da1d9a0d5c7cac73ceae752a90966dc